ravenframework/CodeInterfaceClasses/SCALE/csasData.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on March 25, 2022
+
+@author: aalfonsi
+"""
+import numpy as np
+import os
+import copy
+import re
+
+class csasData:
+  """
+    Class that parses output of scale output (csas sequences) and collect a RAVEN compatible dict
+  """
+  def __init__(self,filen):
+    """
+      Constructor
+      @ In, filen, str, file name to be parsed
+      @ Out, None
+    """
+    self._data = {}
+    self.filename = filen
+    self.loadData()
+
+  def loadData(self):
+    """
+      Method to load data from csas sqeuences
+      @ In, None
+      @ Out, None
+    """
+    self._data['time'] = [0.]
```

Comment:
why time is zero here? If there is only one value for time, why do you need time in your output?

Reply:
(I thought that if a user uses HistorySets, RAVEN will complain because the pivot parameter (time) is not found. I can remove it if you want)

Reply:
I think that depends on the CSAS code. Will a time-dependent outputs generated by CSAS code? If so, I think the code interface need to be revised in order to correctly process it. I would suggest to remove the time for now. @alfoa

Reply:
@wangcj05 removed